alica_dummy_proxy/src/communication/AlicaDummyCommunication.cpp

```diff
@@ -284,10 +283,12 @@ void AlicaDummyCommunication::tick() {}
 void AlicaDummyCommunication::startCommunication()
 {
     _isRunning = true;
+    s_modContainer.registerModule(this);
 }
 
 void AlicaDummyCommunication::stopCommunication()
 {
+    s_modContainer.deregisterModule(this);
```

Comment:
Leaving the deregister on desconstruction in case stopCommunication doesn't get called